backbone.stickit.js

```diff
@@ -449,7 +449,7 @@
     selector: 'input[type="radio"]',
     events: ['change'],
     update: function($el, val) {
-      $el.filter('[value="'+val+'"]').prop('checked', true);
+      $el.filter('[value="'+val.toString().replace(/"/g, '\\"')+'"]').prop('checked', true);
```

Comment:
Please pull this variable out to the line above. Perhaps with a comment explaining what it is for.